src/templates/aws/advanced.ts

```diff
@@ -112,51 +111,190 @@ export default class Advanced {
+    injectToFile('main.tf', ssmModuleContent, this.options, {
+      insertAfter: '# SSM',
+    })
   }
 
   private applySecurityGroup(): void {
-    copyDir('aws/modules/security_group', 'modules/security_group', this.options)
+    copyDir(
+      'aws/modules/security_group',
+      'modules/security_group',
+      this.options,
+    )
 
     const ssmVariablesContent = dedent`
-    variable "app_port" {
-      description = "Application running port"
-      type        = number
-    }
-    
     variable "nimble_office_ip" {
       description = "Nimble Office IP"
     }\n\n`
     appendToFile('variables.tf', ssmVariablesContent, this.options)
 
     const ssmModuleContent = dedent`
     module "security_group" {
-      source = "../modules/security_group"
-    
+      source = "./modules/security_group"
+
       namespace                   = var.app_name
       vpc_id                      = module.vpc.vpc_id
       app_port                    = var.app_port
       private_subnets_cidr_blocks = module.vpc.private_subnets_cidr_blocks
-    
+
       nimble_office_ip = var.nimble_office_ip
     }
     `
 
-    injectToFile('main.tf', ssmModuleContent, this.options, {insertAfter: '# Security groups'})
+    injectToFile('main.tf', ssmModuleContent, this.options, {
+      insertAfter: '# Security groups',
+    })
+  }
+
+  private applyEcr(): void {
+    copyDir('aws/modules/ecr', 'modules/ecr', this.options)
+
+    const ecrVariablesContent = dedent`
+    variable "image_limit" {
+      description = "Sets max amount of the latest develop images to be kept"
+      type        = number
+    }\n\n`
+    appendToFile('variables.tf', ecrVariablesContent, this.options)
+
+    const ecrModuleContent = dedent`
+    module "ecr" {
+      source = "./ecr"
+
+      namespace   = var.app_name
+      owner       = var.owner
+      image_limit = var.image_limit
+    }
+    `
+
+    injectToFile('main.tf', ecrModuleContent, this.options, {
+      insertAfter: '# ECR',
+    })
+  }
+
+  private applyLog(): void {
+    copyDir('aws/modules/log', 'modules/log', this.options)
+
+    const logModuleContent = dedent`
+    module "log" {
+      source = "./modules/log"
+
+      namespace = var.app_name
+    }
+    `
+
+    injectToFile('main.tf', logModuleContent, this.options, {
+      insertAfter: '# Log',
+    })
+  }
+
+  private applyAlb(): void {
+    copyDir('aws/modules/alb', 'modules/alb', this.options)
+
+    const albVariablesContent = dedent`
+    variable "health_check_path" {
+      description = "Application health check path"
+      type = string
+    }
+
+    variable "domain" {
+      description = "Application domain"
+      type        = string
+    }\n\n`
+    appendToFile('variables.tf', albVariablesContent, this.options)
+
+    const albModuleContent = dedent`
+    module "alb" {
+      source = "./modules/alb"
+
+      vpc_id             = module.vpc.vpc_id
+      namespace          = var.app_name
```

Comment:
as mentioned before, we should either use a variable named `app_name` or `namespace` accross this file 🙈